UnityProject/Assets/Scripts/Items/Others/SupermatterSliver.cs

```diff
@@ -0,0 +1,66 @@
+using UnityEngine;
+
+namespace Items
+{
+	public class SupermatterSliver : MonoBehaviour, IServerInventoryMove, ICheckedInteractable<HandApply>
+	{
+		[SerializeField]
+		private ItemTrait supermatterScalpel = null;
+
+		[SerializeField]
+		private ItemTrait supermatterTongs = null;
+
+		public bool vaporizeWhenPickedUp = true;
+		private PlayerScript player;
```

Comment:
```suggestion
```
oh sorry i didnt actually check why codacy was complaining about this, thought it was a different issue. You dont need this.